packages/backend/src/formatter.ts

```diff
@@ -0,0 +1,56 @@
+import { Explore, fieldId as getFieldId, getFields, ResultRow } from 'common';
+
+function formatValue(format: string, value: any): any {
+    switch (format) {
+        case 'km':
+        case 'mi':
+            return `${value} ${format}`;
+        case 'usd':
+            return `$${value}`;
+        case 'gbp':
+            return `${value}£`;
+        case 'percent':
+            return `${parseFloat(value) * 100} %`;
```

Comment:
Might be personal preference but I was really surprised to see a space before the percent sign. Should it be `20%` not `20 %` ?